app/guards/confirmed_user_guard.rb

```diff
@@ -0,0 +1,11 @@
+class ConfirmedUserGuard < Clearance::SignInGuard
+  def call
+    if !signed_in?
+      next_guard
+    elsif current_user.email_confirmed_at.present?
+      next_guard
```

Comment:
Both conditions are needed to show the correct error message. If condensed into just one (e.g. `signed_in? && current_user.email_confirmed_at.present?`), we would always show the confirmation error, even when the issue was wrong username/password.